src/vm/methodtablebuilder.cpp

```diff
@@ -1484,6 +1484,27 @@ MethodTableBuilder::BuildMethodTableThrowing(
         }
     }
 
+#if defined(_TARGET_X86_) || defined(_TARGET_AMD64_) 
+    // All the funtions in System.Runtime.Intrinsics.X86 are hardware intrinsics.
+    // We specially treat them here to reduce the disk footprint of mscorlib.
+    if (GetModule()->IsSystem())
+    {
+        LPCUTF8 x86className;
+        LPCUTF8 x86nameSpace;
+        bmtProp->fIsHardwareIntrinsic = false;
```

Comment:
This is not necessary - all bmtProp fields are zero initialized.

Reply:
Done.